components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/DeviceManagementServiceProviderImpl.java

```diff
@@ -571,4 +571,14 @@ public Operation getOperation(int operationId) throws OperationManagementExcepti
         }
         return devicesOfRole;
     }
+
+    @Override
+    public int getDeviceCount() throws DeviceManagementException {
+        try {
+            int deviceCount = this.deviceDAO.getDeviceCount();
```

Comment:
Fixed in commit [1]
[1] https://github.com/wso2/carbon-device-mgt/commit/00807e8295c06fc4241caebcc514319ae0d88f3b